src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
```

Comment:
I think we can make this shorter, let's get robot value in `for` loop and then use something like this:
![image](https://github.com/mate-academy/js_inverse_robot/assets/96138503/d87ee0d1-a787-4200-a673-258e3736af34)